samples/image-metadata/src/components/FocalPointDialog/FocalPointDialog.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Subheading, Modal } from '@contentful/forma-36-react-components';
+
+import { FocalPoint } from '../FocalPoint';
+import { ImagePreviewWithFocalPoint } from '../ImagePreviewWithFocalPoint';
+
+import { MAX_PREVIEW_WRAPPER_SIZE, styles } from './styles';
+
+export class FocalPointDialog extends Component {
+  static propTypes = {
+    file: PropTypes.object.isRequired,
+    focalPoint: PropTypes.shape({
+      x: PropTypes.number.isRequired,
+      y: PropTypes.number.isRequired
+    }),
+    onClose: PropTypes.func.isRequired,
+    onSave: PropTypes.func.isRequired,
+    sdk: PropTypes.object.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.imgRef = React.createRef();
+  }
+
+  state = {
+    focalPoint: this.props.focalPoint || { x: 0, y: 0 },
```

Comment:
should we default to the center of the image here?

Reply:
Done